events/message/afk-timer.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    name: 'afk-timer',
+    type: 'non-command',
+    aliases: ['example'],
```

Comment:
The AFK timer doesn't want an alias, right? Aliases are for alternames for a command, so i.e. !example 1 2 is same as !args-info 1 2